src/shared/config/config.env.ts

```diff
@@ -5,9 +5,10 @@ import { HostConfig } from './host.config.js';
 import { ItsLearningConfig } from './itslearning.config.js';
 import { LdapConfig } from './ldap.config.js';
 import { PrivacyIdeaConfig } from './privacyidea.config.js';
+import { SystemConfig } from './system.config.js';
 import { OxConfig } from './ox.config.js';
 
-export default (): {
+export type Config = {
     DB: Partial<DbConfig>;
```

Comment:
Good idea to split this off, but I don't think we need to export this type

Reply:
i need the type for the unit tests